src/Phinx/Db/Adapter/BambooHRMysqlAdapter.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Phinx\Db\Adapter;
+
+class BambooHRMysqlAdapter extends \Phinx\Db\Adapter\MysqlAdapter {
+
+	public function hasTable($tableName) {
+		$sql = sprintf("DESC %s", $tableName);
+		try {
+			$exists = $this->query($sql);
+		} catch (\PDOException $exc) {
+			return false;
+		}
+		return true;
+	}
+
+	protected function getForeignKeys($tableName) {
+		throw new \Exception("uh uh - we aren't going to support getting foreign keys");
+	}
```

Comment:
Can we do some regex here to allow for the functionality? Maybe something like this?

```php
protected function getForeignKeys($tableName) {
	$sql = sprintf("SHOW CREATE TABLE '%s'", $tableName);
	$results = $this->query($sql);
	
	$foreignKeyMatch = '/CONSTRAINT\s+\`([a-z\_]*)\`\s+FOREIGN KEY\s+\(([^\)]+)\)\s+REFERENCES\s+([^\(^\s]+)\s*\(([^\)]+)\)/mi';
	preg_match_all($foreignKeyMatch, $results, $rows);
	unset($rows[0]);
	$count = count($rows[1]) - 1;
	$tick = 0;
	$foreignKeys = [];
	while ($tick <= $count) {
		$foreignKeys[$rows[1][$tick]]['table'] = 'esignature_template_fields';
		$foreignKeys[$rows[1][$tick]]['column'] = $rows[2][$tick];
		$foreignKeys[$rows[1][$tick]]['referenced_table'] = $rows[3][$tick];
		$foreignKeys[$rows[1][$tick]]['referenced_column'] = $rows[4][$tick];
		$tick++;
	}
	return $foreignKeys;
}
```